src/prerenderContent/generateHtml.js

```diff
@@ -1,11 +1,17 @@
-export default function generateHtml({
+const assetManifest = fetch("asset-manifest.json").then(response =>
+  response.json()
+);
+
+export default async function generateHtml({
   objId,
   htmlAttributes,
   headContent,
   bodyAttributes,
   bodyContent,
   preloadDumpFileName,
 }) {
+  const assets = await assetManifest;
```

Comment:
* [ ] FYI This code may be a bit more complicated than it needs to be. We're using a top-level const only once, so inlining would be good enough. Also we're mixing `then` and `await`.

Reply:
> Also we're mixing then and await.

If I rewrite the code to this:

```ts
const assetManifestResponse = await fetch("asset-manifest.json");
const assetManifest = await assetManifestResponse.json();
```

Then `eslint` throws a parsing error:
```
Can not use keyword 'await' outside an async function
```

Reply:
> We're using a top-level const only once, so inlining would be good enough.

It's true, that the const is only used once inside of `generateHtml`. But `generateHtml` is called multiple times. If we inline the code into `generateHtml` we would have to fetch the `asset-manifest.json` over and over again.

Reply:
I see. We could still avoid `then` by moving the get-JSON part into the function. But it will not make much of a difference, I guess.